server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -626,7 +626,7 @@ exports.redirectTopologyChangeInformation = async function (body, user, xCorrela
  * customerJourney String Holds information supporting customer’s journey to which the execution applies
  * no response value expected for this operation
  **/
-exports.registerYourself = async function (body, user, xCorrelator, traceIndicator, customerJourney, operationServerName) {
+exports.registerYourself = async function (body, user, xCorrelator, traceIndicator, customerJourney, operationServerName, forwardingName = undefined) {
```

Comment:
variable name shall be more specific like "beaqueathYourDataAndDieForwardingName" instead of forwardingName.
Also include JSdoc describing the purpose of the newly introduced variable.

Reply:
done